scale/job/models.py

```diff
@@ -1495,24 +1052,15 @@ def get_v6_input_data_json(self):
 
         return rest_utils.strip_schema_version(convert_data_to_v6_json(self.get_input_data()).get_dict())
 
-    # TODO: deprecated in favor of get_input_data(), remove this when all uses of it have been removed
+    # # TODO: deprecated in favor of get_input_data(), remove this when all uses of it have been removed
     def get_job_data(self):
```

Comment:
Is get_job_data still used somewhere?

Reply:
It's used quite a few places. `get_input_data` returns a `data.data.Data` object whereas `get_job_data` returns a variant of `JobData` which has more functionality. Not sure if `get_input_data` was supposed to be expanded upon and never was or if all uses of `JobData` were intended to be switched to `data.data.Data`. At this point it's easier to leave the `JobData` than it is to remove all uses.